strider/util.py

```diff
@@ -85,23 +81,8 @@ async def post_json(url, request):
             url,
             json=request,
         )
-        if response.status_code >= 300:
-            raise KPError(
-                "{}\n{}\n\x1b[31m{} Error:\x1b[0m {}".format(
-                    url,
-                    str(request),
-                    response.status_code,
-                    response.text,
-                )
-            )
-        LOGGER.debug(
-            "%s\n%s\n\x1b[32m%d Success:\x1b[0m %s",
-            url,
-            str(request),
-            response.status_code,
-            response.text,
-        )
-    return response.json()
+        response.raise_for_status()
```

Comment:
Does this obscure the error details that may come from the KP?

Reply:
No it does not. The details are still accessible using error.response.text.

Reply:
Can we add a test to ensure that the body of e.g. a 500 response from a KP is included in our log?

Reply:
That is already included in the test: https://github.com/ranking-agent/strider/pull/41/files#diff-c80b6c5e90d69579be550bf44a0a9d5f2ea9d4337d3397fa54e535a72107cdf2R187

Reply:
It seems to me this confirms that it is a 500 response, but not that any further details are captured.

Reply:
This checks that we have captured the text in the response from the server, in this case "Internal server error". This can be verified by changing the text in the decorator for the response overlay and will cause the test to fail. Are there any other details you had in mind to capture?

Reply:
Ah, got it. Thanks for walking me through that.